crates/notary/client/src/client.rs

```diff
@@ -201,6 +204,11 @@ impl NotaryClient {
         notarization_request: NotarizationRequest,
     ) -> Result<(S, String), ClientError> {
         let http_scheme = if self.tls { "https" } else { "http" };
+        let path = if self.path.is_empty() || self.path.starts_with('/') {
```

Comment:
nit: good idea with the 2nd condition, but i prefer to be stricter with our api design, to have simpler logic and avoid confusion with too many patterns (esp. if the upside isn't significant) — i'd only keep the first condition

Reply:
Agree, even if `/` does already exist, most servers will change resulting `//` into `/` - I have checked and PSE server already does